openshift/README.md

```diff
@@ -155,6 +155,19 @@ oc rollout -n $NAMESPACE latest dc/<buildname>-master
 
 *Note: Remember to swap out the bracketed values with the appropriate values!*
 
+## Vanity URL Redirects
+
+In the event a vanity *.gov.bc.ca URL is required for "easier" access to a specific form, we will need some form of reverse-proxy to handle 302 redirects on our behalf. At this time, we are leveraging Caddy to achieve this, specifically with the `bcgov/s2i-caddy:v1-stable` image on Openshift. Note that this will only need to be performed in the prod environment once, as this does not apply to any other environments. In order to deploy this reverse proxy, run the following (where APP_NAME should match with the public name and intended path such as `minesoperatorscreening`):
+
+```sh
+export NAMESPACE=<yournamespace>
+export APP_NAME=<yourappshortname>
+
+oc process -n $NAMESPACE -f caddy-reverse-proxy.dc.yaml APP_NAME=$APP_NAME -o yaml | oc apply -n $NAMESPACE -f -
+```
+
+This template will implement a deploymentconfig, a configmap with the appropriate caddyfile settings, and an attached service. A route must be manually created to expose the service after the template has been deployed.
```

Comment:
why aren't we creating a route?

Reply:
At this time we don't know if we need a route with a custom SSL certificate or not. If it does, this can't be templated at all and must be manually inserted.

Reply:
well, we know we are going to need a route.  so it would be better to stand one up, then, if a cert cannot be added to the route after the fact, you just create the one with the cert and drop the auto-created one.  you don't lose anything by creating it, but you do by not creating it.

Reply:
We can add in a route object easily enough when we have confirmation on how we would be handling vanity URLs correctly. I have manually inserted a route through the web console in the meantime.